Microsoft/WebTemplateStudio
# dandua98/keybord nav

This PR adds Keyboard Navigation to the wizard, therefore making all inputs, buttons, links and all the functionality exposed by Web Template Studio accessible using just the keyboard! 

_**Testing Notes:**_

- Launch Web Template Studio in dev mode using VSCode (by opening `src/extension` and pressing F5)
- Try navigating through the whole wizard using just your keyboard. Verify the ability to deploy Azure services and generate an app using the keyboard alone. _Note: Use Tab and Enter to navigate~


Closes #318
